.github/workflows/develop.yml

```diff
@@ -103,6 +103,13 @@ jobs:
         run: |
           ./meta-dts/scripts/generate-ipxe-menu.sh ${{steps.dts-ver.outputs.DTS_VER}}
           scp -i ~/.ssh/dts-ci-key dts-rc.ipxe builder@10.1.40.2:boot/dts/
+      - name: Deploy manifest on boot.dasharo.com
```

Comment:
Move this before `Deploy sha256 on boot.dasharo.com` and generate `sha256` for manifest also?

Reply:
https://github.com/Dasharo/meta-dts/compare/11eca457865d78c9d63b0964397f5d3d121d3682..f7a9a5e7b3901cb6445a630203c323ca4e9821bc

Reply:
There is so much code duplication in those workflows :(